### ISSUE TITLE
Handle a lot of creators better

### ISSUE THREAD

**Issue by dandlezzz** [holaplex/marketplace#167]
URS and other gallery style marketplaces could have over 50 creators. We need a better way to display them and it would be good to start thinking of a way to distinguish "Creators" from upcoming "Collections".

**Comment by damianr** [comment]
@echohtp yes for sure! I think we'll need to support both adding all from a creator and the collections standard. We have a ticket to index the collections and that will deff be the most reliable and flexible way to add NFTs to a marketplace but in the short term we need to handle creators better.

**Comment by damianr** [comment]
https://www.figma.com/file/eACyDSbpcWYoE9ypOZNlf8/Marketplaces?node-id=1008%3A27178

**Comment by alchemistgo87** [comment]
<img width="226" alt="Screenshot 2022-04-08 at 4 57 31 PM" src="https://user-images.githubusercontent.com/1254443/162426905-b8a1de23-775e-488c-b645-2101d69ce243.png">

@damianr I have queries regarding the nft card:
- Do we have to hardcode Creator here?
- What link to open on click of Creator if an nft has multiple creators?
- Which image to show with creator here (in case of multiple creators)?

**Comment by damianr** [comment]
Good questions @alchemistgo87. 

The creators we should show on the NFT card are only the ones that have been added to the marketplace. And they should link to the creator page on the marketplace not the profile. 

For NFTs with multiple creators let's do this. 
<img width="290" alt="image" src="https://user-images.githubusercontent.com/260165/162491917-428b1e8f-cc01-4614-9cc0-84569839b62f.png">

**Comment by alchemistgo87** [comment]
@damianr 

Here, which page will be opened with a click of multiple creators' images. Should each of these small images open up the respective creator page? If so, it would be hard for user to point to each small image.

**Comment by alchemistgo87** [comment]
@damianr One more question - Do we show both carousel as well as 'See all' button for creators' preview on the home page?

**Comment by alchemistgo87** [comment]
@kespinola Should we add stats and profile data to StoreCreator (marketplace) to fetch total nfts and twitter profile in this new creators' preview design:

![Screenshot 2022-04-11 at 5 21 08 PM](https://user-images.githubusercontent.com/1254443/162734164-5fa153ec-92ff-41b1-9a45-db246b7b9e06.png)

**Comment by kespinola** [comment]
> @kespinola Should we add stats and profile data to StoreCreator (marketplace) to fetch total nfts and twitter profile in this new creators' preview design:

@alchemistgo87 yes but we need make sure they are done with data loaders and need limit on the creators query to only grab 3 at a time. hitting the > should set the offset to 3 to show the next 3. limit always 3.

**Comment by alchemistgo87** [comment]
Thanks @kespinola, I will keep these points in mind!

**Comment by kespinola** [comment]
done.
